p2p/src/rate_limit.rs

```diff
@@ -1,224 +0,0 @@
-// Copyright 2016 The Grin Developers
```

Comment:
Are we just getting rid of the rate limit stuff for now?
I don't think its a bad idea to simplify this for now (was never sure what would happen if we actually hit the rate limit mid message anyway).

Reply:
Yes, I decided to leave it out for now. It seemed a bit premature and a limit in message count for each message type may serve us better if it comes to that.